### ISSUE TITLE
REQ Phone version - Firmware - modules

### ISSUE THREAD

**Issue by rocketrobot99** [alucas/Snorlax#34]
Hi,

I am in a position to be able to buy a phone just for development purposes. Ideally, I would like to get one that will have little or no issues running POGO with root and Xposed / rootswitch. Is there any chance we can create a list of people's working solutions. ATM I have an SM-G900F Samsung S5 that I cannot get to work.

Maybe the author could start off with their make model etc..

Thanks in advance, I really want to start using this...

**Comment by alexcheveau** [comment]
All working fine here. My setup is:

Nexus 6p
Stock Android 6.0.1
Twrp 3.0.2-3
Systemless Xposed 86.2 sdk23 topjohnwu
SuperSU 2.78 SR4
Suhide 0.55
Rootswtich 1.3.1.2

**Comment by alucas** [comment]
Motorola G4
I think I used the sames tool as @alexcheveau

**Comment by Dem0nIO** [comment]
LG e975
Resurrection Remix 5.7.4
TWRP
Systemless Root
Systemless Xposed
Root Switch

**Comment by mhetralla** [comment]
That would be a long list. Basically, it should work right now on every phone where you can install **Systemless SuperSU**, **suhide**, **Systemless Xposed 86.2** and **RootSwitch**. That is every phone that is AOSP or AOSP-like, and it doesn't have the November patch.

**Comment by rocketrobot99** [comment]
@Dem0nIO Thanks everyone, I now have an E975 with resurrection Remix 5.7.4. However still struggling to get Xposed working. I can get it installed in TWRP. but it says it is not installed. Plus using rootswitch, pokemon Go can't log in. Do you have the exact versions of the various bits you used? Thanks.

**Comment by Dem0nIO** [comment]
@rocketrobot99 the versions i have installed are:

ResurrectionRemix-M-v5.7.4-20160925-geehrc
TRWP 3.0.2-0
Systemless SuperSu SR1-SuperSU-v2.79-SR1-20161221223537
SUHIDE 0.55
Systemless Xposed xposed-v86.2-sdk23-topjohnwu
Xposed Installer APK 3.0.1 by dvdandroid
Root Switch 1.3.3.2

Check if your TRWP version is the same as i am using. Ill be happy to answer if you have more questions.

**Comment by rocketrobot99** [comment]
I can't find TWRP 3.0.2-0 for geehrc anywhere, all the links are broken. I hope it won;t make a difference and will try flashing the other bits.
*Update* Doesn't work, I can't seem to flash xposed, it just comes up with an error '! Failed: Unable to exract zip file' If I modify the installer-script It looks like it installs, but when I reboot it says not installed. 
Thanks for your help anyway @Dem0nIO

**Comment by Dem0nIO** [comment]
What about your bootloader @rocketrobot99?
Mine bootloader version is MAKOZ30f, what is yours?
I would kindly share a google drive with twrp 3.0.2-0 and MAKOZ30f if you need them

**Comment by rocketrobot99** [comment]
Yes, the bootloader is the same, can you share both though just in case please? @Dem0nIO

**Comment by Dem0nIO** [comment]
There @rocketrobot99 As soon as you get them I will delete the files. Also, if you need more assitance, i have setup a discord

edit: deleted links

**Comment by rocketrobot99** [comment]
still not working. xposed installs ok, but is not recognised. In root switch what is the selinux policy on your phone? mine switches between permissive and enforcing, I don't think it should. I may start from scratch, can I just flash a stock rom?

**Comment by Dem0nIO** [comment]
@rocketrobot99 That is what Root Switch does aside from disabling root. When you enable it, goes from Permissive to Enforcing. You do not need to go back to stock. did you unroot after installing RRemix?
I recommend that you uninstall root switch, Xposed, Suhide, SuperSu. Of course, it will be easier if you just reflash RRemix 5.7.4, then install gapps, then unroot and see if you can pass safetynet...

**Comment by rocketrobot99** [comment]
I have success today! Many thanks. I am used to samsung phones where I just delete /system/bin/su and /system/xbin/su, I am not used to 'unroot'ing. Xposed still says it is not installed, however snorlax works fine.
